### DISCUSSION TITLE
v0.17.4-beta.rc1 release notes link is broken

### DISCUSSION THREAD

**Discussion by reivanen** [lightningnetwork/lnd#8458] [General]
The release notes link in https://github.com/lightningnetwork/lnd/releases/tag/v0.17.4-beta.rc1 is broken.

is: https://github.com/lightningnetwork/lnd/blob/v0.17.4-beta/docs/release-notes/release-notes-0.17.4.md

fixed: https://github.com/lightningnetwork/lnd/blob/v0.17.4-beta.rc1/docs/release-notes/release-notes-0.17.4.md

**Comment by yyforyongyu** [comment]
Updated, thanks!

**Comment by hieblmi** [comment]
This seems resolved.
